_docs/getting-started-with-wdl.md

```diff
@@ -109,16 +109,21 @@ runtime {
 }
 ```
 
-Finally, we have a metadata section where we can store key value pairs. It is freeform, so we could put anything here. Dockstore is able to pick up both Author and Email if they are defined here. In this example we only have the Author.
+Finally, we have a metadata section where we can store key value pairs. It is freeform, so we could put anything here. Dockstore is able to pick up author, email, and description if they are defined here. 
+The description field can be used to add documentation to your workflow and can make use of markdown syntax. Currently, description values are limited to single line string, but multi-line documentation is possible by including new line characters into the string. 
```

Comment:
Maybe add another sentence, which unfortunately is getting into the details of markdown syntax, that since a blank line is significant in markdown, you will need two `\n` to represent a blank line.

IOW, there are two points you are making, and I conflated them with my proposal:

1. In general, if you want a new line in your string, you have to represent it with `\n`.
2. For markdown, blank lines can be significant and so you will need two newlines to get a blank line in between two lines.

It would be good if this were captured in your example.

Reply:
:+1: